src/tests/test_dte_data_models.py

```diff
@@ -1617,24 +1763,26 @@ def test_validate_referencias_rut_otro_is_consistent_with_emisor_rut(self) -> No
 
         expected_validation_errors = [
             {
-                'loc': ('__root__',),
-                'msg': "'rut_otro' must be different from 'emisor_rut':"
-                " Rut('60910000-1') == Rut('60910000-1'),"
-                " 'Referencia' number 1.",
+                'loc': (),
+                'msg': "Value error, 'rut_otro' must be different from 'emisor_rut': "
+                "Rut('60910000-1') == Rut('60910000-1'), 'Referencia' number 1.",
```

Comment:
Reformat to reduce diff size.

```suggestion
                'msg': (
                    "Value error, "
                    "'rut_otro' must be different from 'emisor_rut':"
                    " Rut('60910000-1') == Rut('60910000-1'),"
                    " 'Referencia' number 1."
                ),
```